src/rpc/mining.cpp

```diff
@@ -472,7 +472,9 @@ static RPCHelpMan prioritisetransaction()
         throw JSONRPCError(RPC_INVALID_PARAMETER, "Priority is no longer supported, dummy argument to prioritisetransaction must be 0.");
     }
 
-    EnsureMemPool(request.context).PrioritiseTransaction(hash, nAmount);
+    if (!EnsureMemPool(request.context).PrioritiseTransaction(hash, nAmount)) {
+        throw JSONRPCError(RPC_MISC_ERROR, "PrioritiseTransaction failed.");
```

Comment:
Maybe a bit more verbosity here would be good.

Reply:
Sure, if we can find an error message that will stay true over time :)  What is your suggestion?

Reply:
Now using a slightly more verbose error message which hints that an invalid fee_delta argument is the most likely cause. Let me know if you have any other suggestion on wording.